homeassistant/components/juicenet/sensor.py

```diff
@@ -117,22 +112,4 @@ def icon(self):
     @property
     def state(self):
         """Return the state."""
-        state = None
-        sensor_type = self.entity_description.key
-        if sensor_type == "status":
-            state = self.device.status
-        elif sensor_type == "temperature":
-            state = self.device.temperature
-        elif sensor_type == "voltage":
-            state = self.device.voltage
-        elif sensor_type == "amps":
-            state = self.device.amps
-        elif sensor_type == "watts":
-            state = self.device.watts
-        elif sensor_type == "charge_time":
-            state = self.device.charge_time
-        elif sensor_type == "energy_added":
-            state = self.device.energy_added
-        else:
-            state = "Unknown"
-        return state
+        return getattr(self.device, self.entity_description.key, None)
```

Comment:
```suggestion
        return getattr(self.device, self.entity_description.key, "Unknown")
```
To match the previous behavior

Reply:
`None` was good (probably better even), just noting that the behavior changed :smiley: